plugins/opsgenie/icon_opsgenie/util/api.py

```diff
@@ -1,12 +1,22 @@
+from logging import Logger
+import time
+from typing import Callable, Type
 import requests
 
 from insightconnect_plugin_runtime.exceptions import PluginException
 
 from .validators import InputDataValidator
 
+RETRY_MESSAGE = "Rate limiting error occurred. Retrying in {delay} seconds ({attemps_counter}/{max_tries})"
```

Comment:
```suggestion
RETRY_MESSAGE = "Rate limiting error occurred. Retrying in {delay} seconds ({attempts_counter}/{max_tries})"
```
Fix for the variable name - this will need to be updated for all usages